org.eclipse.mylyn-target/org.eclipse.mylyn.target.target

```diff
@@ -79,5 +94,30 @@
 			<unit id="org.mockito.mockito-core" version="0.0.0"/>
 			<unit id="org.objenesis" version="0.0.0"/>
 		</location>
+		<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" label="Apache Lucene" missingManifest="generate" type="Maven">
```

Comment:
@merks this is what I found regarding Apache Lucene in [Orbit](https://github.com/eclipse-orbit/orbit-simrel/blob/9b6c913a1987119983ff4fbc62c1c4e3ec5f8e1a/maven-bnd/tp/MavenBND.target#L67)